examples/ct/ctmapper/mapper/mapper.go

```diff
@@ -82,7 +82,7 @@ func (m *CTMapper) oneMapperRun(ctx context.Context) (bool, error) {
 	}
 
 	mapperMetadata := &ctmapperpb.MapperMetadata{}
-	if getRootResp.MapRoot.Metadata != nil {
+	if getRootResp.GetMapRoot().Metadata != nil {
```

Comment:
Just `GetMapRoot()` doesn't buy you much, as panics anyway with a (hypothetically) nil getRootResp.MapRoot. We either go all the way (`getRootResp.GetMapRoot().GetMetadata()`) or don't.